src/actions/project-text-search.js

```diff
@@ -35,6 +36,12 @@ export function clearSearchResults() {
   };
 }
 
+export function clearSearch() {
+  return ({ dispatch, getState }: ThunkArgs) => {
```

Comment:
Since this only fires the one action you can remove the wrapping `return ({ dispatch, getState }: ThunkArgs) => {...}` and return just the object.